pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -164,27 +206,36 @@ configuration in Istio.
 #traceSamplingDuration=0
 ## Always sample spans with an error tag (set to true) ignoring other sampling configuration. Defaults to true.
 #traceAlwaysSampleErrors=false
-## A comma separated, list of additional custom tag keys to include along as metric tags for the
-## derived RED(Request, Error, Duration) metrics. Applicable to Jaeger and Zipkin integration only.
-#traceDerivedCustomTagKeys=tenant, env, location
 
-## The following settings are used to configure histogram ingestion:
+########################################## HISTOGRAM ACCUMULATION SETTINGS #############################################
 ## Histograms can be ingested in wavefront scalar and distribution format. For scalar samples ports can be specified for
-## minute, hour and day granularity. Granularity for the distribution format is encoded inline.
-## Before using any of these settings, reach out to Wavefront Support to ensure your account is enabled for native Histogram
-## support and to optimize the settings for your specific use case.
+## minute, hour and day granularity. Granularity for the distribution format is encoded inline. Before using any of
+## these settings, reach out to Wavefront Support to ensure your account is enabled for native Histogram support and
+## to optimize the settings for your specific use case.
+
+## Accumulation parameters
+## Directory for persisting proxy state, must be writable.
+#histogramStateDirectory=/var/spool/wavefront-proxy
+## Interval to write-back accumulation changes to disk in millis (only applicable when memory cache is enabled)
```

Comment:
Interval to write-back accumulation changes to disk in millis...
=> Interval to write back accumulation changes to disk, in ms 
(or: in milliseconds. but not in millis)